data/mods/gen8bdsp/formats-data.ts

```diff
@@ -227,7 +227,7 @@ export const FormatsData: {[k: string]: ModdedSpeciesFormatsData} = {
 	persian: {
 		tier: "RU",
 		doublesTier: "DUU",
-		randomBattleMoves: ["fakeout", "slash", "uturn", "bite", "aerialace"],
+		randomBattleMoves: ["fakeout", "covet", "uturn", "bite", "aerialace"],
```

Comment:
Is this really better? Is Persian without an item so often that it's worth giving up the extra damage from Slash?

Reply:
Covet gets boosted by Technician

Reply:
Oh, right, I forgot about Technician. Sure, that's an improvement then.